samtranslator/internal/schema_source/sam-docs.json

```diff
@@ -408,21 +408,29 @@
-      "Config": "Configure the authorization provider that you specify in `Type`\\. This property supports the following types:  \n+ Amazon Cognito user pools \u2013 `AMAZON_COGNITO_USER_POOLS`\n+ OpenID Connect \u2013 `OPENID_CONNECT`\n+ Lambda authorizer \u2013 `AWS_LAMBDA`\n*Type*: [LambdaAuthorizerConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appsync-graphqlapi-lambdaauthorizerconfig.html) \\| [OpenIDConnectConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appsync-graphqlapi-openidconnectconfig.html) \\| [UserPoolConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appsync-graphqlapi-userpoolconfig.html)  \n*Required*: Conditional\\. Required for the types previously mentioned\\.  \n*AWS CloudFormation compatibility*: Depending on your `Type`, this property is passed directly to the [`LambdaAuthorizerConfig`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlapi.html#cfn-appsync-graphqlapi-lambdaauthorizerconfig), [`OpenIDConnectConfig`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlapi.html#cfn-appsync-graphqlapi-openidconnectconfig), or [`UserPoolConfig`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlapi.html#cfn-appsync-graphqlapi-userpoolconfig) property of an `AWS::AppSync::GraphQLApi` resource\\.",
-      "Type": "The default authorization provider between applications and your AWS AppSync GraphQL API\\.  \nFor a list and description of allowed values, see [Authorization and authentication](https://docs.aws.amazon.com/appsync/latest/devguide/security-authz.html) in the *AWS AppSync Developer Guide*\\.  \nWhen you specify a Lambda authorizer \\(`AWS_LAMBDA`\\), AWS SAM creates an AWS Identity and Access Management \\(IAM\\) policy to provision permissions between your GraphQL API and Lambda function\\.  \n*Type*: String  \n*Required*: Yes  \n*AWS CloudFormation compatibility*: This property is passed directly to the [`AuthenticationType`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlapi.html#cfn-appsync-graphqlapi-authenticationtype) property of an `AWS::AppSync::GraphQLApi` resource\\."
+      "Additional": "A list of additional authorization types for your GraphQL API\\.  \n*Type*: List of [ AuthProvider](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-graphqlapi-auth-authprovider.html)  \n*Required*: No  \n*AWS CloudFormation compatibility*: This property is unique to AWS SAM and doesn't have an AWS CloudFormation equivalent\\.",
```

Comment:
We should also start using these in the models so that they appear in the final schema. Specifically, add assignments to https://github.com/aws/serverless-application-model/blob/develop/samtranslator/internal/schema_source/aws_serverless_graphqlapi.py in similar fashion to https://github.com/aws/serverless-application-model/blob/develop/samtranslator/internal/schema_source/aws_serverless_function.py.